models/repo.go

```diff
@@ -695,7 +705,7 @@ func MigrateRepository(u *User, opts MigrateRepoOptions) (*Repository, error) {
 		repo.DefaultBranch = headBranch.Name
 	}
 
-	return repo, UpdateRepository(repo, false)
+	return repo, nil
```

Comment:
Should we call `UpdateRepository` here and looks more consist because normally you wouldn't expect an update operation after such function?